api/src/auth.rs

```diff
@@ -99,7 +99,7 @@ impl UserDirectory {
     fn from_user_file() -> Self {
         let file_path = Self::users_toml_file_path();
         let file_contents: String = std::fs::read_to_string(&file_path)
-            .expect(&format!("this should blow up if the users.toml file is not present at {:?}", &file_path));
+            .unwrap_or_else(|_| panic!("this should blow up if the users.toml file is not present at {:?}", &file_path));
```

Comment:
I'm curious to why replace expect with this construct.

Reply:
Ahh this is again [a Clippy lint](https://rust-lang.github.io/rust-clippy/master/#expect_fun_call) :) Using `unwrap_or_else` avoids the unnecessary call to `format!` should `read_to_string` return `Ok`.